src/pages/Pool/index.test.tsx

```diff
@@ -0,0 +1,46 @@
+import * as useV3Positions from 'hooks/useV3Positions'
+import { BrowserRouter as Router } from 'react-router-dom'
+import { render, screen } from 'test-utils'
+import * as switchChain from 'utils/switchChain'
+
+import Pool from '.'
+
+jest.mock('@web3-react/core', () => {
+  const web3React = jest.requireActual('@web3-react/core')
+  return {
+    ...web3React,
+    useWeb3React: () => ({
+      chainId: 1,
+    }),
+  }
+})
+
+describe('networks', () => {
+  it('renders error card when unsupported chain is selected', () => {
+    jest.spyOn(switchChain, 'isChainAllowed').mockReturnValue(false)
+    jest.spyOn(useV3Positions, 'useV3Positions').mockImplementation(() => {
+      return { loading: false, positions: undefined }
+    })
+
+    render(
+      <Router>
+        <Pool />
+      </Router>
+    )
+    expect(screen.getAllByTestId('pools-unsupported-err')).toHaveLength(1)
```

Comment:
better to check for the text to display rather than checking for the id

Reply:
screen.getByText wasn't working for me here because there was another element (the link) embedded into the Trans the displayed text was in. I messed around for a while trying to match the text and link but ended up going with the id, can definitely change it if there is a proper way to use getByText